isos/appliance.sh

```diff
@@ -124,6 +124,7 @@ chroot $(rootfs_dir $PKGDIR) chmod 0660 /var/run/lock/logrotate_run.lock
 cp ${BIN}/vic-init $(rootfs_dir $PKGDIR)/sbin/vic-init
 
 cp ${BIN}/{docker-engine-server,port-layer-server,vicadmin} $(rootfs_dir $PKGDIR)/sbin/
+cp ${BIN}/unpack $(rootfs_dir $PKGDIR)/bin/
```

Comment:
will probably move this back to line 126 and put it in /sbin with everything else now that I have the execution worked out